apps/dock/dock.py

```diff
@@ -13,9 +13,33 @@ class Actions:
     def dock_send_notification(notification: str):
         """Send a CoreDock notification to the macOS Dock using SPI"""
 
+    def dock_app_expose():
+        """Activate app Exposé using the macOS Dock item for the frontmost app"""
+
 
 @ctx.action_class("user")
 class UserActions:
+    def dock_app_expose():
+        from pathlib import Path
```

Comment:
Why not something like this?

```suggestion
    def dock_app_expose(app: Optional[App]):
        if app is None:
            app = ui.active_app()
```

Than a user could choose to add:

```<user.running_applications> window expose user.dock_app_expose(running_applications) ```

I've generally taken this pattern in my own scripts rather than hardcoding `ui.active_app()` inside of it; it makes the action more flexible (even if we don't ship this particular grammar in community, although I think it's useful)

Reply:
`user.running_applications` returns a `str` rather than a `ui.App`. But this works:

```
^window <user.running_applications>$:
    app = user.get_running_app(running_applications)
    user.dock_app_expose(app)
```

Seems OK?

Reply:
Oops, correct.